### PR TITLE
Fix support for BIGINT in SPL params

### LINKED ISSUE #387: Type LONG is not supported
We have an issue with testing Teradata stored procedures that have BIGINT data type as out parameter.
"Execute procedure" fixture throws exception:
java.lang.UnsupportedOperationException: TeradataEnvironment: getSqlType: Type LONG is not supported.
I believe there are set of Teradata database data types that treated as Java LONG data type.
It would be good to have these data types supported too.

### PR DESCRIPTION
**Pull request by MMatten**
Corrected type mappings to fix #387.

### CODE REVIEW

**Comment by MMatten** [comment]
Just noticed I need to rebase this onto the updated `master`...

**Comment by MMatten** [comment]
Rebase done.

File: FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This stored proc tests output parameters - CalcLength returns string length as second output param
 
-|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength numeric)
+|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength bigint)
```
Comment (javornikolov) [inline FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt]: As we're removing the `numeric` version here - do we have some other test which is still covering the `numeric` behavior?

Comment (MMatten) [inline FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt]: We're still covered in `StoredProcWithException`.

None of the data type tests for Teradata stored price and functions are explicit enough really. I wonder if we could define some standard tests sections and use `!include`s to supply the specific content for each DB sometime.

Comment (javornikolov) [inline FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt]: OK. Seems good enough to me for now.

**Comment by javornikolov** [comment]
:+1 I'm merging this. Thank you @MMatten for the quick fix!

**Pull request merged**